replication/row_event_test.go

```diff
@@ -0,0 +1,69 @@
+package replication
+
+import (
+	"testing"
+)
+
+func TestDecodeDecimal(t *testing.T) {
```

Comment:
We should use gocheck instead of origin testing.T like other test cases in go-mysql. :-)